notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -55,10 +55,10 @@
    "source": [
```

Comment:
<p><code>nncf.compress_weights</code> function accept =&gt; <code>nncf.compress_weights</code> function accept<strong>s</strong> OpenVINO model instance and compress<strong>es</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done</p>